api/v1alpha1/databasesecretengineconfig_types.go

```diff
@@ -151,7 +159,6 @@ type DBSEConfig struct {
 
 	// VerifyConnection Specifies if the connection is verified during initial configuration. Defaults to true.
 	// +kubebuilder:validation:Optional
-	// +kubebuilder:default=true
```

Comment:
please leave this here, otherwise the default will be false.

Reply:
It doesn't work if I leave it there. Even with `verifyConnection: false` in a supplied YAML, the custom resource created by the test has `verifyConnection: true` in it, and the controller sends `verify_connection: true` in the payload to Vault. If I `kubectl apply` the YAML file, the created custom resource has `verifyConnection: false` in it, which leads me to believe that there's a bug somewhere else in the existing integration test code, but I couldn't find it.

Without the `default=true`, the YAML value is respected. Also FWIW, if you omit the argument altogether, Vault assumes it to be `true`. RabbitMQ secret engine's config works the same way: https://github.com/redhat-cop/vault-config-operator/blob/14b154211ca0f4515715c92d334c6f64b656e4da/api/v1alpha1/rabbitmqsecretengineconfig_types.go#L67-L69